sentry/src/main/java/io/sentry/SentryFeedbackOptions.java

```diff
@@ -0,0 +1,543 @@
+   */
+  public boolean isEmailRequired() {
+    return isEmailRequired;
+  }
+
+  /**
+   * Sets whether the email field on the feedback form is required. Defaults to false.
+   *
+   * @param isEmailRequired true if the email field is required
+   */
+  public void setEmailRequired(boolean isEmailRequired) {
+    this.isEmailRequired = isEmailRequired;
+  }
+
+  /**
+   * Displays the email field on the feedback form. Ignored if isEmailRequired is true. Defaults to
+   * true.
+   *
+   * @return true if the email field is shown
+   */
+  public boolean isShowEmail() {
+    return showEmail;
+  }
+
+  /**
+   * Sets whether the email field on the feedback form is shown. Ignored if isEmailRequired is true.
+   * Defaults to true.
+   *
+   * @param showEmail true if the email field is shown
+   */
+  public void setShowEmail(final boolean showEmail) {
+    this.showEmail = showEmail;
+  }
+
+  /**
+   * Sets the email and name fields to the corresponding Sentry SDK user fields that were called
+   * with SentrySDK.setUser. Defaults to true.
+   *
+   * @return true if the email and name fields are set to the Sentry SDK user fields
+   */
+  public boolean isUseSentryUser() {
+    return useSentryUser;
+  }
+
+  /**
+   * Sets whether the email and name fields are set to the corresponding Sentry SDK user fields that
+   * were called with SentrySDK.setUser. Defaults to true.
+   *
+   * @param useSentryUser true if the email and name fields are set to the Sentry SDK user fields
+   */
+  public void setUseSentryUser(final boolean useSentryUser) {
+    this.useSentryUser = useSentryUser;
+  }
+
+  /**
+   * Displays the Sentry logo inside of the form. Defaults to true.
+   *
+   * @return true if the Sentry logo is shown
+   */
+  public boolean isShowBranding() {
+    return showBranding;
+  }
+
+  /**
+   * Sets whether the Sentry logo is shown inside of the form. Defaults to true.
+   *
+   * @param showBranding true if the Sentry logo is shown
+   */
+  public void setShowBranding(final boolean showBranding) {
+    this.showBranding = showBranding;
+  }
+
+  /**
+   * The title of the feedback form. Defaults to "Report a Bug".
+   *
+   * @return the title of the feedback form
+   */
+  public @NotNull String getFormTitle() {
+    return formTitle;
+  }
+
+  /**
+   * Sets the title of the feedback form.
+   *
+   * @param formTitle the title of the feedback form
+   */
+  public void setFormTitle(final @NotNull String formTitle) {
+    this.formTitle = formTitle;
+  }
+
+  /**
+   * The label of the submit button. Defaults to "Send Bug Report".
+   *
+   * @return the label of the submit button
+   */
+  public @NotNull String getSubmitButtonLabel() {
+    return submitButtonLabel;
+  }
+
+  /**
+   * Sets the label of the submit button. Defaults to "Send Bug Report".
+   *
+   * @param submitButtonLabel the label of the submit button
+   */
+  public void setSubmitButtonLabel(final @NotNull String submitButtonLabel) {
```

Comment:
I'm thinking it'd be a nice DevX to have overloads that accept `@StringRes int resId` like [this](https://cs.android.com/android/platform/superproject/main/+/main:frameworks/base/core/java/android/widget/TextView.java;l=7613-761).

Also this one (and other Strings) should probably be changed to a `CharSequence` to make it more versatile

Reply:
i will change the options to `CharSequence`